domains/elora.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "elorahub",
+        "email": "elorahubph@gmail.com"
+    },
+    "record": {
+        "A": "162.159.205.11",
+        "CNAME": "apex-loadbalancer.netlify.com"
```

Comment:
You can not use CNAME with other records

Reply:
Hello I already done updating, thank you inadvance